src/Composition/responseCalculation/elasticCollisionResponse.jl

```diff
@@ -36,22 +36,22 @@ function resultantDampingCoefficient(cor, abs_vreln, vsmall, maximumContactDampi
     return d_res
 end
 
-function elasticContactPairCoefficients(obj1::Object3D, obj2::Object3D)
-    solid1::Shapes.Solid = obj1.feature
-    solid2::Shapes.Solid = obj2.feature
-
+function elasticContactPairCoefficients(obj1::Object3D{F}, obj2::Object3D{F}) where F <: Modia3D.VarFloatType
+    solid1::Shapes.Solid{F} = obj1.feature
+    solid2::Shapes.Solid{F} = obj2.feature
+    mu_r_geo::F = F(0.0)
```

Comment:
Type definition is not necessary.

Reply:
Ok, I'd need to re-run this.

Reply:
To my understanding, this type definitions are needed, e.g. obj.feature is an abstract type, therefore ::Shapes.Solid{F} is needed. mu_r_geo::F is also needed, because F(0.0) converts 0.0 to F, but the returned type of the conversion function is an Any type, as I understand, and then mu_r_geo::F is needed.

Reply:
I tried it out: ` mu_r_geo = F(0.0)`  is fine